programs/avocado/meta.yaml

```diff
@@ -0,0 +1,35 @@
+description: >
+ a set of tools and libraries to help with automated testing.
+ One can call it a test framework with benefits. Native tests are written in Python
+ and they follow the unittest pattern, but any executable can serve as a test.
+
+homepage: https://github.com/avocado-framework
+
+parameters:
+ test_group:
+
+results:
```

Comment:
no need to have results for this functional test

Reply:
ok